### ISSUE TITLE
TamperMonkey v4.19.0 expires in the ungoogled-chromium_131.0.6778.204-1.1

### ISSUE THREAD

**Issue by Casuwin** [ungoogled-software/ungoogled-chromium#3134]
### OS/Platform

Windows

### Installed

https://ungoogled-software.github.io/ungoogled-chromium-binaries/

### Version

ungoogled-chromium_131.0.6778.204-1.1

### Have you tested that this is not an upstream issue or an issue with your configuration?

- [ ] I have tried reproducing this issue in Chrome and it could not be reproduced there
- [ ] I have tried reproducing this issue in vanilla Chromium and it could not be reproduced there
- [ ] I have tried reproducing this issue in ungoogled-chromium with a new and empty profile using `--user-data-dir` command line argument and it could not be reproduced there

### Description

TamperMonkey v4.19.0 expires in the ungoogled-chromium_131.0.6778.204-1.1

### How to Reproduce?

1. download ungoogled-chromium_131.0.6778.204-1.1.exe from https://ungoogled-software.github.io/ungoogled-chromium-binaries/releases/windows/64bit/131.0.6778.204-1
2. installl ungoogled-chromium_131.0.6778.204-1.1.exe
3. run the new version ungoogled-chromium

### Actual behaviour

The Tampermonkey icon is displayed in the extension bar, but when clicked, the window shows no scripts. However, when entering the Tampermonkey management panel, all scripts are still marked as running, but they do not display in the popup window and cannot run on the corresponding website either.
All other extensions displayed in the extension bar are functioning normally; only the scripts in Tampermonkey cannot run.

### Expected behaviour

none

**Comment by Harfho** [comment]
are you sure , its caused by the update?
a check on the lastest commit of chromium_131.0.6778.204.1 shows they only changes the version number 
[](https://github.com/ungoogled-software/ungoogled-chromium/compare/131.0.6778.139-1...131.0.6778.204-1)

**Comment by Casuwin** [comment]
Yes, I am sure. In fact, the same issue still occurs with version 131.0.6778.204, so I am currently using version 130.0.6723.116.

**Comment by Casuwin** [comment]
Before I switched to version 131.0.6778.204.1, I was using version 129.0.6668.100.

**Comment by PF4Public** [comment]
Is this still an issue?
